CPT331.Data.Migration/Migrations/Crime.spGetCrimeOffenceLocalGovernmentAreaState.sql

```diff
@@ -0,0 +1,61 @@
+﻿CREATE PROCEDURE Crime.spGetCrimeOffenceLocalGovernmentAreaState
+
+@Skip INT = 0,
+@Take INT = 100,
+@OrderBy VARCHAR(10) = 'ID',
+@SortDirection VARCHAR(4) = 'ASC'
+
+AS
+
+IF (@OrderBy IS NULL) OR (@OrderBy = '')
+BEGIN
+	SET @OrderBy = 'ID'
+END
+
+IF (@SortDirection IS NULL) OR (@SortDirection = '')
+BEGIN
+	SET @SortDirection = 'ASC'
+END
+
+SELECT
+	ID,
+	StateID,
+	StateName,
+	LocalGovernmentAreaID,
+	LocalGovernmentAreaName,
+	OffenceID,
+	OffenceName,
+	[Count],
+	[Month],
+	[Year],
+	IsDeleted,
+	IsVisible,
+	DateCreatedUtc,
+	DateUpdatedUtc
+FROM
+	Crime.CrimeOffenceLocalGovernmentAreaState
+
+ORDER BY
+	ID
+	/*
+	CASE WHEN @OrderBy = 'Count' AND @SortDirection = 'ASC' THEN [Count] END ASC,
```

Comment:
Regardless, I spent a little bit of time getting this to work, and it wasn't very quick, so I commented it out and though that perhaps someone smarter than me can fix it in a later PR. I'm looking at you, @6nop!